hls4ml/convert.py

```diff
@@ -181,8 +192,34 @@ def main(args):
             plt.ylabel('QKeras {}'.format(klayer))
             plt.savefig(os.path.join(cfg['OutputDir'], 'profiling_{}.png'.format(layer)), dpi=300)
     else:
+        # Original save of input_data/output_preds
+        np.savetxt('input_data.dat', X_test[:1].reshape(1, -1), fmt='%f', delimiter=' ')
+        np.savetxt('output_predictions.dat', y_test[:1].reshape(1, -1), fmt='%f', delimiter=' ')
+    
+        # Extending input_data/output_pred by appending
+        # the data/preds of more test images (num_test_imgs)
+        for i in range(1, num_test_imgs):
+            with open('input_data.dat', "ab") as f:
+                np.savetxt(f, X_test[i].reshape(1, -1), fmt='%f', delimiter=' ')
+    
+            with open('output_predictions.dat', "ab") as f:
+                np.savetxt(f, y_test[i].reshape(1, -1), fmt='%f', delimiter=' ')
```

Comment:
do we still need this? i was under the impression that if you look in `tb_data/tb_input_features.dat` and `tb_data/tb_output_predictions.dat` the inputs/outputs are already put there based on whatever numpy arrays are input in this configuration: https://github.com/hls4ml-finn-mlperftiny/CIFAR10/blob/04ec51260e0931a9aca8cc283f4f793ce6792a6a/hls4ml/convert.py#L138-L139